Makefile

```diff
@@ -77,13 +77,18 @@ chaosdaemon: generate fmt vet
 manager: generate fmt vet
 	$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaos-controller-manager ./cmd/controller-manager/*.go
 
+# Build chaosfs binary
 chaosfs: generate fmt vet
 	$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaosfs ./cmd/chaosfs/*.go
 
+# Build chaoscm binary
+chaoscm: generate fmt vet
+	$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaoscm ./cmd/chaoscm/main.go
+
 dashboard: fmt vet
 	$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaos-dashboard ./cmd/chaos-dashboard/*.go
 
-binary: chaosdaemon manager chaosfs dashboard
+binary: chaosdaemon manager chaosfs chaoscm dashboard
```

Comment:
what means chaoscm? chaos CPU & Memory?

Reply:
Yes

Reply:
Maybe we should call them `StressChaos` to include CPU, MEM, FS, IO and so on. And init a stress framework for it.

Reply:
Yes. It is better

Reply:
OK. I will set up a stress framework and add cpu, memory along with it.